Jasmine/LevelDesigner/ViewControllers/LevelDesignerViewController.swift

```diff
@@ -35,17 +41,19 @@ class LevelDesignerViewController: UIViewController {
 
     /// Feeds in the appropriate data for the use of seguing into this view.
     ///
-    /// - Parameter viewModel: the level designer view model required to use this view
-    func segueWith(_ viewModel: LevelDesignerViewModel) {
+    /// - Parameters:
+    ///   - viewModel: the level designer view model required to use this view
+    ///   - onSaveCallBack: Callback to be executed after saving
+    func segueWith(_ viewModel: LevelDesignerViewModel, onSaveCallBack: @escaping () -> Void) {
```

Comment:
implement in level selector vc: 
levelDesigner.onSaveCallback = {
    display custom levels 
}